src/display/misc/keybinds.go

```diff
@@ -40,8 +40,15 @@ var procKeybindings = []string{
 	"  - 0: Disable Sort",
 	"",
 	"[Process actions](fg:white)",
-	"[Requires confirmation, press key again to confirm](fg:white)",
-	"  - K and <F9>: Kill process",
+	"  - K and <F9>: Open signal selector menu",
+	"",
+	"[Signal selection](fg:white)",
+	"  - K and <F9>: Send SIGTERM to selected process",
```

Comment:
(nitpick) maybe mention that this kills the process. People not familiar with unix signals may not understand this.

Reply:
Makes sense. Will do